deployment/docker/compose-vt-single.yml

```diff
@@ -0,0 +1,119 @@
+services:
+
+  hotrod:
+    image: jaegertracing/example-hotrod-snapshot:461923b41ff10964d723e8578421710d0ef42c3f
+    ports:
+      - "8080:8080"
+      - "8083:8083"
+    command: [ "all" ]
+    environment:
+      - OTEL_EXPORTER_OTLP_TRACES_ENDPOINT=http://victoriatraces:9428/insert/opentelemetry/v1/traces
+    depends_on:
+      - victoriatraces
+
+  # Grafana instance configured with VictoriaLogs as datasource
+  grafana:
+    image: grafana/grafana:11.5.0
+    depends_on:
+      - "victoriametrics"
+      - "victoriatraces"
+    ports:
+      - 3000:3000
+    volumes:
+      - grafanadata:/var/lib/grafana
+      - ./provisioning/datasources/victoriametrics-traces-datasource/single.yml:/etc/grafana/provisioning/datasources/single.yml
+      - ./provisioning/dashboards:/etc/grafana/provisioning/dashboards
+      - ./provisioning/plugins/:/var/lib/grafana/plugins
+      - ./../../dashboards/victoriametrics.json:/var/lib/grafana/dashboards/vm.json
+      - ./../../dashboards/victoriatraces.json:/var/lib/grafana/dashboards/vl.json
+    restart: always
+
+  #  VictoriaTraces instance, a single process responsible for
+  #  storing trace span and serving read queries.
+  victoriatraces:
+    # todo: use official image when it is released
+    image: 'docker.io/victoriametrics/victoria-logs:trace-test-v0.0.3'
+    ports:
+      - "9428:9428"
+    command:
+      - "--storageDataPath=/vtraces"
+    volumes:
+      - vtdata:/vtraces
+
+  # VictoriaMetrics instance, a single process responsible for
+  # scraping, storing metrics and serve read requests.
+  victoriametrics:
+    image: victoriametrics/victoria-metrics:v1.119.0
+    ports:
+      - "8428:8428"
+    volumes:
+      - vmdata:/storage
+      - ./prometheus-vt-single.yml:/etc/prometheus/prometheus.yml
+    command:
+      - "--storageDataPath=/storage"
+      - "--promscrape.config=/etc/prometheus/prometheus.yml"
+    restart: always
+
+  # vmauth is a router and balancer for HTTP requests.
+  # It proxies query requests from vmalert to either VictoriaMetrics or VictoriaLogs,
+  # depending on the requested path.
+  vmauth:
+    image: victoriametrics/vmauth:v1.119.0
+    depends_on:
+      - "victoriametrics"
+      - "victoriatraces"
+    volumes:
+      - ./auth-vt-single.yml:/etc/auth.yml
+    command:
+      - "--auth.config=/etc/auth.yml"
+    ports:
+      - 8427:8427
+    restart: always
+
+  # vmalert executes alerting and recording rules according to the given rule type.
```

Comment:
I don't think we need vmalert here at all. 
I'd recommend keeping traces example as simple as possible. It doesn't need to be as complete as vlogs/vmetrics examples.
If we get rid of vmalert, we can also remove vmauth and alertmanager. Making all the setup much more lightweight

Reply:
Alternatively, you can remove vtraces-example-alerts.yml because they don't make sense. If you do so, you can get rid of vmauth too.

Reply:
Big plus for getting rid of vmalert and related stuff

Reply:
Thank you for the suggestion. Let's keep the demo minimal with the following components only:
1. Application (HotROD)
2. VictoriaTraces
3. VictoriaMetrics
4. Grafana

Reply:
And add alerting when we have well-designed alerting with traces (it's available not tho, as VictoriaLogs provide such functionality, but not tested)